### PR TITLE
daily coding practice 10  is added

### PR DESCRIPTION
**Pull request by waghmaresuyog**

### CODE REVIEW

File: simple/src/codingPractice/DivideString.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        String[] equalStr = new String[numberOfPart];
+        if (length % numberOfPart != 0) {
+            System.out.println("Sorry this string cannot be divided into " + numberOfPart + " equal parts.");
+        } else {
+
+            for (int index = 0; index < length; index = index + chars) {
+                String part = stringComplete.substring(index, index + chars);
+                equalStr[temporary] = part;
+                temporary++;
+
```
Comment (Ams12) [inline simple/src/codingPractice/DivideString.java:19]: Remove extra  spaces

File: simple/src/codingPractice/DivideString.java [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 public class DivideString {
     public static void main(String[] args) {
-
         String stringComplete = "aaaabbbbcccc";
```
Comment (Ams12) [inline simple/src/codingPractice/DivideString.java:5]: Use proper String name.

File: simple/src/codingPractice/DivideString.java (hunk 1 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/DivideString.java]: done

File: simple/src/codingPractice/DivideString.java (hunk 2 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/DivideString.java]: done

File: simple/src/codingPractice/DivideString.java [hunk 3]
```diff
@@ -2,16 +2,16 @@
 
 public class DivideString {
     public static void main(String[] args) {
-        String stringComplete = "aaaabbbbcccc";
-        int length = stringComplete.length();
+        String bunchOfString = "aaaabbbbcccc";
```
Comment (Ams12) [inline simple/src/codingPractice/DivideString.java:5]: Use proper variable value instead of "aaaabbbbcccc".

**Review by Ams12 — APPROVED** [review]

File: simple/src/codingPractice/DivideString.java (hunk 3 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/DivideString.java]: done

**Pull request merged**